terraform/backend/main.tf

```diff
@@ -2,20 +2,34 @@ provider "aws" {
   region = "eu-west-1"
 }
 
+# TODO:tfsec Enable bucket encryption
+# TODO:tfsec Consider enabling access logging
```

Comment:
Is there any reason not to enable bucket encryption now?

Yes you're right that I applied this manually. But I guess we could move it to a module and let test/staging/prod include it themselves now.  Just need to import the existing resources

Reply:
Encryption and logging enabled now :+1:

I think just keep it manual, it feels wrong to import the state bucket into itself.